### PR TITLE
Gh 23

### PR DESCRIPTION
**Pull request by Ross-Alexandra**
Please let me know before merging this. I don't have time to setup a test heroku environment and a test database, so I'm hoping the connection will work correctly, but I'd like to be on standby in case something fails.

### CODE REVIEW

File: ccf/app/settings/base.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+For more information on this file, see
+https://docs.djangoproject.com/en/2.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.1/ref/settings/
+"""
+
+import os
+
+# import django_heroku
```
Comment (eetar1) [inline ccf/app/settings/base.py]: Unnecessary

File: ccf/app/settings/base.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+STATIC_ROOT = os.path.join(BASE_DIR, "staticfiles")
+STATIC_URL = "/static/"
+
+LOGIN_REDIRECT_URL = "/"
+
+# Extra places for collectstatic to find static files.
+STATICFILES_DIRS = (os.path.join(BASE_DIR, "static"),)
+
+# Must be at end of settings.
+# django_heroku.settings(locals())
```
Comment (eetar1) [inline ccf/app/settings/base.py]: do we need this

File: ccf/app/views.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+from django.shortcuts import redirect, render
```
Comment (eetar1) [inline ccf/app/views.py]: rendered could be removed

File: ccf/app/settings/base.py [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+Generated by 'django-admin startproject' using Django 2.1.7.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.1/ref/settings/
+"""
+
+import os
```
[the remark this answers is not in the document]
Comment (Ross-Alexandra) [reply ccf/app/settings/base.py]: Good catch, will remove

File: ccf/app/settings/base.py (hunk 2 above)
Comment (Ross-Alexandra) [reply ccf/app/settings/base.py]: No, will remove.

File: ccf/app/views.py (hunk 3 above)
Comment (Ross-Alexandra) [reply ccf/app/views.py]: Good catch, will remove

File: ccf/app/settings/base.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (Ross-Alexandra) [reply ccf/app/settings/base.py]: Addressed in https://github.com/Ross-Alexandra/SENG-371-Project-2/pull/43/commits/a61859e168304df545112c4360af8cfe1fac8f06

File: ccf/app/settings/base.py (hunk 2 above)
Comment (Ross-Alexandra) [reply ccf/app/settings/base.py]: Addressed in https://github.com/Ross-Alexandra/SENG-371-Project-2/pull/43/commits/a61859e168304df545112c4360af8cfe1fac8f06

File: ccf/app/views.py (hunk 3 above)
Comment (Ross-Alexandra) [reply ccf/app/views.py]: Addressed in https://github.com/Ross-Alexandra/SENG-371-Project-2/pull/43/commits/a61859e168304df545112c4360af8cfe1fac8f06

File: ccf/tests/test_models.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    )
+
+    job.save()
+
+    assert len(Jobs.objects.all()) == 1
+
+
+@pytest.mark.django_db()
+def test_creating_job_without_dockerfile_raises_integrity_error():
+    job = Jobs(finished=False, datastore_link="abc.com")
```
Comment (joshmcintosh) [inline ccf/tests/test_models.py]: What happens if you drop the `finished` argument?

Comment (Ross-Alexandra) [reply ccf/tests/test_models.py]: Has a default, but good call, will test the default

Comment (Ross-Alexandra) [reply ccf/tests/test_models.py]: Addressed in https://github.com/Ross-Alexandra/SENG-371-Project-2/pull/43/commits/4bedbac1592ea65e07e8b93ec5f726d8d950774d

**Pull request merged**